janus_core/helpers/utils.py

```diff
@@ -188,6 +189,21 @@ def spacegroup(
     )
 
 
+def snap_symmetry(struct: Atoms, sym_tolerance: float = 0.001) -> None:
+    """
+    Refine symmetry of structure.
```

Comment:
"Refine" is an odd phrasing for forcing the symmetry

Reply:
I'm borrowing from the ASE, but we can change it if you think force or similar is more explicit

```
def refine_symmetry(atoms, symprec=0.01, verbose=False):
    """
    Refine symmetry of an Atoms object
```